### ISSUE TITLE
Accept head attributes/tags from poi while using ejs templates

### ISSUE THREAD

**Issue by klauskpm** [egoist/poi#310]
**Do you want to request a *feature* or report a *bug*?**
This is a feature request.

**What is the current behavior?**
When I configure the title and description at the poi configuration file, while using an ejs template file, they don't affect the page.

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
I would like that the title and description, as any other head configuration features there are supported, set in the poi configuration file would change the HTML of a ejs file, when it doesn't have a head with those properties.

**If this is a feature request, what is the motivation or use case for changing the behavior?**
I wasted a good amount of time to understand that the head title wasn't updating just because I started using the ejs template file. My template is only changing the body, and yet the HTML with head and body are created.

**Please mention other relevant information such as the browser version, Node.js version, Poi version and Operating System.**
Chrome: Version 63.0.3239.84
NodeJS: v8.9.3
NPM: 5.5.1
Yarn: 1.3.2
Poi: 9.6.1
OS: macOS Sierra version 10.12.5

Here is my repository: https://github.com/klauskpm/poi-testing

**Comment by mblarsen** [comment]
You could copy the default [ejs template](https://github.com/egoist/poi/blob/master/packages/poi/lib/index.ejs) and modify it to your needs. Your repo contains only the app div, so that is all that gets output.

**Comment by mblarsen** [comment]
Here is your app runnning by copying the default template. 

<img width="487" alt="screen shot 2017-12-18 at 10 41 55 am" src="https://user-images.githubusercontent.com/247048/34087745-2342f7d6-e3e0-11e7-8888-dc28134b98bb.png">

**Comment by klauskpm** [comment]
Thanks for the full example. I knew about the htmlWebpackPlugin, but, I would like to have it set by the config file, when I'm only using the ejs for the content.

I don't know if this is a ejs or poi restriction, but it would be nice to have this implementation.

**Comment by mblarsen** [comment]
Do you mean that you would like to only provide the data for the default template (like the title), but not necessarily provide a new template?

In that case, you can just delete your `index.ejs` file and remove `template` from your example code:

```
module.exports = {
    html: {
        title: 'My title is amazingly invisible',
        description: 'My description',
    }
};
```

This will include title and description using the default template.

**Comment by klauskpm** [comment]
No. I want to include the title and description in the ejs file, without using the htmlWebpackPlugin. Doing that in the same way it is done when it is not set a template. Looking at my repo I'm only including a div, and yet a complete HTML is output.

So, it would be nice to have the my poi HTML configuration rendered inside the head tag, when it is automatically generated.

**Comment by mblarsen** [comment]
> and yet a complete HTML is output.

The "complete" output you see is generated by your browser. If you look at the actual content of what is output from your template is only the the app div:

<img width="530" alt="screen shot 2017-12-18 at 8 24 27 pm" src="https://user-images.githubusercontent.com/247048/34106039-a49e8844-e431-11e7-9f01-277606e8c5db.png">

If you look at my comment above I did this after checking out your repo:

1. Deleted index.ejs (so we use default template)
2. Remvoved the `template` key from `package.json`, but kept title and description.

In other words with a clean poi setup you'd only add:

```js
module.exports = {
    html: {
        title: 'My title is amazingly invisible',
        description: 'My description',
    }
};
```

To get what you ask for.

**Comment by egoist** [comment]
@mblarsen I guess what he means is using his own template file but making `title` option always work even if that's not added in his template file.

**Comment by mblarsen** [comment]
To inject it into any arbitrary template?

I wouldn't want that:

1. It is already easy to customize template.
2. If I do customize template I wouldn't want any *magic* happening to it :D 

Little win for more complicated code.

**Comment by klauskpm** [comment]
I see. I was wrong. I thought the complete output was from the application.

Still, what I wanted was to inject into some arbitrary templates. The ones without html, head, and body tags. But I can see the trouble it would make to check every template to inject.

So, I think this feature request should be closed, not because of htmlWebpackPlugin, but the hassle it would be to check the templates to only them inject it.
